buildSrc/src/main/java/co/anitrend/buildSrc/extensions/ProjectExtensions.kt

```diff
@@ -78,6 +86,9 @@ fun Project.hasKaptSupport() =
 fun Project.hasKotlinAndroidExtensionSupport() =
     name != Modules.App.Domain.id
 
+internal val Project.libs: LibrariesForLibs get() =
+    (this as ExtensionAware).extensions.getByName("libs") as LibrariesForLibs
+
```

Comment:
This is a helper function that replicates the `libs` extension function in app modules. See [comment](https://github.com/AniTrend/anitrend-v2/pull/185/files#r1060746927) for more context